src/sidebar/components/test/login-control-test.js

```diff
@@ -1,115 +1,284 @@
 'use strict';
 
 var angular = require('angular');
+var proxyquire = require('proxyquire');
 
+var bridgeEvents = require('../../../shared/bridge-events');
 var util = require('../../directive/test/util');
 
-function PageObject(element) {
-  this.unknownLoginText = function () {
-    return element[0].querySelector('.login-text').textContent;
-  };
-  this.loginText = function () {
-    return Array.from(element[0].querySelectorAll('.login-text a'))
-      .map(function (el) { return el.textContent; });
-  };
-  this.menuLinks = function () {
-    return Array.from(element[0].querySelectorAll('.login-control-menu .dropdown-menu a'))
-      .map(function (el) { return el.textContent; });
-  };
-  this.disabledMenuItems = function () {
-    return Array.from(element[0].querySelectorAll('.login-control-menu .dropdown-menu__link--disabled'))
-      .map(function (el) { return el.textContent.trim(); });
-  };
-  this.menuText = function () {
-    return element[0].querySelector('span').textContent;
+function pageObject(element) {
+  return {
+    menuLinks: function () {
+      return Array.from(element[0].querySelectorAll('.login-control-menu .dropdown-menu a'))
+                  .map(function (el) { return el.textContent; });
+    },
+    menuText: function () {
+      return element[0].querySelector('span').textContent;
+    },
+    userProfileButton: element[0].querySelector('.dropdown-menu__user-profile-btn'),
```

Comment:
To clarify, I'm _only_ proposing to add `js-*` classes _if there is no existing styled class_ that can be used to get a reference to the element. Alternatively, you could use a utility function to find the element based on some other criteria (eg. its text content).